### ISSUE TITLE
Ability to analyze differences in income variance across neighborhoods

### ISSUE THREAD

**Issue by CooperNederhood** [uchicago-computation-workshop/luis_bettencourt#9]
I understand your neighborhood income levels are based on Census Block Group data containing 600 - 3,000 people. Does the Census Block Group data contain income _variance_ to enable analogous analysis on heterogeneity in variance across various neighborhoods? 

In the event such data is not available, do you have any expectations as to possible heterogeneity in neighborhood income variance across the cities you analyzed?

Comparing the variance of middle-income neighborhoods could be particularly useful to distinguish between a true mixture of high/middle/low income households and a homogeneous neighborhood of only middle-income.

**Comment by lmbett** [comment]
Yes, this is somewhat contained in the analysis of the distributions. Ask me more if that is not clear...
